### PR TITLE
feat: update definitions and add more examples to component explorer

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Updated component definitions to include missing slots and attributes.
Updated component examples to include more configurations.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-component-explorer/app/fast-components/configs/fast-anchor.ts [hunk 1]
```diff
@@ -14,6 +20,35 @@ const fastAnchorConfig: ComponentViewConfig = {
...
                         data: {
+                            href: "#",
+                            Slot: [
+                                {
+                                    id: "Slot",
+                                },
+                            ],
+                        },
+                    },
+                    Slot: {
```
Comment (chrisdholt) [inline sites/fast-component-explorer/app/fast-components/configs/fast-anchor.ts:31]: should these all be capitalized?

Comment (eljefe223) [reply sites/fast-component-explorer/app/fast-components/configs/fast-anchor.ts:31]: Yes, this is how Jane defined them in the mappings.

https://github.com/microsoft/fast-dna/blob/763054f36433f9b87e620c0c8d03c229dcd8560f/packages/tooling/fast-tooling/src/data-utilities/mapping.ts#L434

File: sites/fast-component-explorer/app/fast-components/configs/fast-flipper.ts [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 import Guidance from "../../.tmp/flipper/guidance";
+import { placeholderIcon } from "../utilities/icons";
 import { ComponentViewConfig } from "./data.props";
 
+const previousPlaceholderIcon = placeholderIcon("root", "SlotPrevious");
+const nextPlaceholderIcon = placeholderIcon("root", "SlotNext");
+
 export const fastFlipperId = "fast-flipper";
+
+console.log(fastComponentSchemas[fastFlipperId]);
```
Comment (janechu) [inline sites/fast-component-explorer/app/fast-components/configs/fast-flipper.ts:11]: might want to delete this

File: sites/fast-component-explorer/app/fast-components/configs/fast-anchor.ts [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
 import Guidance from "../../.tmp/anchor/guidance";
+import { placeholderIcon } from "../utilities/icons";
 import { ComponentViewConfig } from "./data.props";
 
+const startPlaceholderIcon = placeholderIcon("root", "SlotStart");
+const endPlaceholderIcon = placeholderIcon("root", "SlotEnd");
+const defaultPlaceholderIcon = placeholderIcon("root", "Slot");
+
 export const fastAnchorId = "fast-anchor";
+console.log(fastComponentSchemas[fastAnchorId]);
```
Comment (janechu) [inline sites/fast-component-explorer/app/fast-components/configs/fast-anchor.ts:11]: leftover?

File: sites/fast-component-explorer/app/fast-components/configs/fast-switch.ts [hunk 4]
```diff
@@ -1,8 +1,9 @@
-import { fastComponentSchemas } from "@microsoft/site-utilities";
+import { fastComponentSchemas, textSchema } from "@microsoft/site-utilities";
 import Guidance from "../../.tmp/switch/guidance";
 import { ComponentViewConfig } from "./data.props";
 
 export const fastSwitchId = "fast-switch";
+console.log(fastComponentSchemas[fastSwitchId]);
```
Comment (janechu) [inline sites/fast-component-explorer/app/fast-components/configs/fast-switch.ts:6]: leftover console log

Comment (eljefe223) [reply sites/fast-component-explorer/app/fast-components/configs/fast-switch.ts]: Whoops!

**Pull request merged**